oauth/_auth.md

```diff
@@ -11,14 +14,16 @@ import {
 
 export const AuthProcess: AuthSpecification = {
   Identifier: {
+
+// DECLARE THE APP RELIES ON OAUTH
     AuthenticationType: AuthenticationType.OAuth,
+
     IsSandbox: false,
   },
   ...
 };
 
 export const Metadata: OrderSourceAppMetadata = {
-  // DO NOT CHANGE THIS ID AFTER PUBLISHING
```

Comment:
removed on purpose?

Reply:
Sorta. I started to remove all of the "context" from the code samples, as it seemed like they distracted from what it was trying to explain. But then I had second thoughts and resorted most of it, because I can see some value in demonstrating where in the metadata something belongs. Really wish I could highlight a specific line with a different background color. I think I'll restore this line for now.